webpack.config.js

```diff
@@ -1,42 +1,56 @@
-const path = require('path')
-const HtmlWebpackPlugin = require('html-webpack-plugin')
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-    entry: './src/index.js',
-    output: {
-        path: path.join(__dirname, '/dist'),
-        filename: 'bundle.js'
-    },
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                exclude: /\node_modules/,
-                use: {
-                    loader: 'babel-loader'
-                }
-            },
-            {
-                test: /\.css$/,
-                use: [
-                    'style-loader',
-                    'css-loader'
-                ]
+  entry: ['babel-polyfill', './src/index.js'],
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /\node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+      {
+        test: /\.css$/,
+        use: [
+          {
+            loader: 'style-loader',
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              modules: {
+                localIdentName: '[local]--[hash:base64:5]',
+              },
             },
-            {
-                test: /\.(png|jpg|gif)$/i,
-                use: [
-                  {
-                    loader: 'url-loader'                    
-                  },
-                ],
-            }
-        ]
+          },
+        ],
+      },
+      {
+        test: /\.(png|jpg|gif)$/i,
+        use: [
+          {
+            loader: 'url-loader',
+          },
+        ],
+      },
+    ],
+  },
+  resolve: {
+    alias: {
+      src: path.resolve(__dirname, 'src'),
+      Components: path.resolve(__dirname, 'src/components'),
     },
-    plugins: [        
-        new HtmlWebpackPlugin({
-        template: './src/index.html'
-        })
-    ]
-
-}
\ No newline at end of file
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
```

Comment:
`index.html` should be moved to public folder (need to be created)

Reply:
It was updated